index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const extractDir = path.resolve(__dirname, 'extract');
+const headlessDir = path.join(extractDir, 'headless_shell');
+const pathToNss = path.join(extractDir, 'nss', fs.readFileSync(path.join(extractDir, 'nss', 'latest'), 'utf8').trim());
+
+exports.chromePath = headlessDir;
+exports.additionalBinaryPaths = [path.join(pathToNss, 'lib')];
+exports.additionalLibraryPaths = [path.join(pathToNss, 'bin')];
```

Comment:
These two seem switched (`binary` points to `lib` and vice versa)

Reply:
Just a rough idea: should we also export a tiny function that will patch the environment of current process with the things above? (e.g. append `additionalLibraryPaths` to `LD_LIBRARY_PATH`, etc.) What do you think?

Reply:
Good catch.

I had been reluctant to patch the environment, preferring to pass it in when chrome is spawned. However, it's a lot simpler to patch so you've tipped the balance.